### PR TITLE
Parametrize rule for login.defs hashing algorithm

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Parametrize rule `set_password_hashing_algorithm_logindefs`.
  - Allow choice between `SHA512` and `SHA256`.
- Add simple tests.

#### Rationale:

- More flexibility for the rule.

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-pam/var_password_hashing_algorithm.var [hunk 1]
```diff
@@ -0,0 +1,17 @@
+documentation_complete: true
+
+title: Password Hassing algorithm
+
+description: |-
+    Specify the hashing algorithm.
```
Comment (redhatrises) [inline linux_os/guide/system/accounts/accounts-pam/var_password_hashing_algorithm.var:6]: Can you flush this description out a bit more?

File: linux_os/guide/system/accounts/accounts-pam/var_password_hashing_algorithm.var [hunk 2]
```diff
@@ -0,0 +1,17 @@
+documentation_complete: true
+
+title: Password Hassing algorithm
```
Comment (mildas) [inline linux_os/guide/system/accounts/accounts-pam/var_password_hashing_algorithm.var:3]: `title: Password Hashing Algorithm`

Comment (yuumasato) [reply linux_os/guide/system/accounts/accounts-pam/var_password_hashing_algorithm.var]: Should be fixed, thanks.

**Comment by mildas** [comment]
LGTM

**Pull request merged**